draft/real-world-phoenix_rendering-markdown.md

```diff
@@ -0,0 +1,133 @@
+
+```elixir
+# config/config.exs
+
+config :phoenix, :template_engines,
+  md: PhoenixMarkdown.Engine,
+  scrambled: RealWorldPhoenix.Engines.Scrambled
+```
+
+So I am expecting this to scramble content written in markdown. I'm just considering paragraphs of text, just to not complicate things unnecessarily for this use case. That is why I am basically splitting sentences by splitting on the newline character and then processing each sentence and the containing words before joining them together again with newlines. Seems pretty straightforward. Let's see if this works!
+
+## Smartypants... 🤔
+
+Phoenix Markdown has an option to render server_tags, which basically means that you can invoke Elixir inside your markdown page using the standard tags you use in `eex` templates to invoke some Elixir code and evaluate it. I wanted to use this to render the following bit of text to illustrate my `scrambled` engine, but the first pass led to this error while compiling:
+
+```bash
+== Compilation error in file lib/real_world_phoenix_web/views/post_view.ex ==
+** (SyntaxError) lib/real_world_phoenix_web/templates/post/blogs/2020-01-28/rendering_markdown.html.md:60: unexpected token: "“" (column 39, code point U+201C)
+    (eex) lib/eex/compiler.ex:45: EEx.Compiler.generate_buffer/4
+    (phoenix) lib/phoenix/template.ex:361: Phoenix.Template.compile/3
+    (phoenix) lib/phoenix/template.ex:167: anonymous fn/4 in Phoenix.Template."MACRO-__before_compile__"/2
+    (elixir) lib/enum.ex:1948: Enum."-reduce/3-lists^foldl/2-0-"/3
+    (phoenix) expanding macro: Phoenix.Template.__before_compile__/1
+    lib/real_world_phoenix_web/views/post_view.ex:1: RealWorldPhoenixWeb.PostView (module)
+    (elixir) lib/kernel/parallel_compiler.ex:229: anonymous fn/4 in Kernel.ParallelCompiler.spawn_workers/7
+```
+What? 😕🤷
+
+Ok, so what is happening here. It is seeing an unexpected token and it seems to be a left handed double quote. After banging my head against the wall for about a day, I decided to `rtfm` again and luckily Boyd has us covered. See [here](https://github.com/boydm/phoenix_markdown#unexpected-token-in-server-tags), and I'll quote it below:
+
+_By default Earmark replaces some characters with prettier UTF-8 versions. For example, single and double quotes are replaced with left- and right-handed versions. This may break any server tag which contains a prettified character since EEx cannot interpret them as intended. To fix this, disable smartypants processing._
+
+So it is actually a simple fix in our config:
+```elixir
+config :phoenix_markdown, :earmark, %{
+  smartypants: false
+}
+```
+
+<%= Phoenix.LiveView.Helpers.live_render(@conn, RealWorldPhoenixWeb.Live.Scrambled) %>
```

Comment:
This part is actually rendering a LiveView when you view this post on realworldphoenix.com 
I'll probably at that in as static text for the guild